### ISSUE TITLE
[Accessibility Audit] Batch Changes: Configure personal user credentials for publishing changesets with Batch Changes

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#34073]
### Steps to replicate journey

Taken from the [user journey list document](https://docs.google.com/document/d/1XgG0udYL37G3RxuT5shizulpHgb6xVtY5L9kBeSkXYk/edit?usp=sharing). Use this for further context.

- Click your user profile avatar in the top right corner
- Click “Settings” from the dropdown menu
- Click “Batch Changes” from the sidebar, within the “Account” section
- Click “Add credentials” for a code host without credentials configured (has a red empty circle next to the code host name)
- Follow the steps to [create a new personal access token on the code host](https://docs.sourcegraph.com/batch_changes/how-tos/configuring_credentials#creating-a-code-host-token)
  - Note: We cannot guarantee the WCAG status of external sites, we should aim to just ensure that the instructions are accessible.
- Paste the personal access token into the input field
- Click “Add credential”
- Wait for the modal to close
- A green checkbox should appear next to the code host name

### How to audit

Follow the instructions here: [Auditing a user journey](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#auditing-a-user-journey)

**Note:** We are planning on arranging thorough third-party audit in the future, so our focus here is just to find the _majority_ of accessibility issues. Don't worry if you feel you haven't found 100% of potential issues, it is better to focus on the core essentials to complete the journey rather than spending lots of time going through every possible problem.

### How to raise discovered problems

Follow the instructions here: [Raising an accessibility bug](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#raising-a-bug)

**Comment by courier-new** [comment]
Audit complete and issues filed.
